apps/docs/src/app/platform/component-docs/platform-forms/platform-checkbox-group/platform-checkbox-group-docs.component.html

```diff
@@ -0,0 +1,41 @@
+<fd-docs-section-title [id]="'def'" [componentName]="'PlatformCheckboxGroupListComponent'">
```

Comment:
You are using the same `[id]` attribute for every documentation section.

Reply:
fixed.